packages/shared/src/components/modals/post/CommentModal.tsx

```diff
@@ -0,0 +1,254 @@
+import React, {
```

Comment:
Code in this file is mostly copied over from the `CommentInputPage.tsx`. And adapted to work in a modal.

Reply:
Can you apply the recent changes on the `CommentInputPage` to utilize the new Context for writing a comment. This is to fix the state of the buttons on the header along with keyboard shortcuts.